apps/testingaccessibility/studio/schemas/documents/review.js

```diff
@@ -37,6 +43,16 @@ export default {
         },
       ],
     },
+    {
+      name: 'body',
+      type: 'body',
+      title: 'Page Body',
+    },
```

Comment:
we have a custom body type which is an object with portable text and some customization around it.